x-pack/plugins/stack_connectors/server/connector_types/cases/jira/service.ts

```diff
@@ -498,8 +501,9 @@ export const createExternalService = (
   };
 
   const getIssues = async (title: string) => {
+    const modifiedTitle = escapeSpecialCharacters(title ?? '');
```

Comment:
nit: There is no need to do `?? ''`.

Reply:
fixed in [a73197](https://github.com/elastic/kibana/pull/145610/commits/a73197ef9e85657180c33cd6cee4290f9b8e655a)